testproject/Packages/manifest.json

```diff
@@ -1,48 +1,48 @@
 {
-    "dependencies": {
-      "com.unity.collab-proxy": "1.2.16",
-      "com.unity.ide.rider": "1.1.4",
-      "com.unity.ide.vscode": "1.2.3",
-      "com.unity.multiplayer.mlapi": "file:../../com.unity.multiplayer.mlapi",
-      "com.unity.package-validation-suite": "0.19.2-preview",
-      "com.unity.test-framework": "1.1.19",
-      "com.unity.textmeshpro": "2.1.1",
-      "com.unity.timeline": "1.2.17",
-      "com.unity.ugui": "1.0.0",
-      "com.unity.modules.ai": "1.0.0",
-      "com.unity.modules.androidjni": "1.0.0",
-      "com.unity.modules.animation": "1.0.0",
-      "com.unity.modules.assetbundle": "1.0.0",
-      "com.unity.modules.audio": "1.0.0",
-      "com.unity.modules.cloth": "1.0.0",
-      "com.unity.modules.director": "1.0.0",
-      "com.unity.modules.imageconversion": "1.0.0",
-      "com.unity.modules.imgui": "1.0.0",
-      "com.unity.modules.jsonserialize": "1.0.0",
-      "com.unity.modules.particlesystem": "1.0.0",
-      "com.unity.modules.physics": "1.0.0",
-      "com.unity.modules.physics2d": "1.0.0",
-      "com.unity.modules.screencapture": "1.0.0",
-      "com.unity.modules.terrain": "1.0.0",
-      "com.unity.modules.terrainphysics": "1.0.0",
-      "com.unity.modules.tilemap": "1.0.0",
-      "com.unity.modules.ui": "1.0.0",
-      "com.unity.modules.uielements": "1.0.0",
-      "com.unity.modules.umbra": "1.0.0",
-      "com.unity.modules.unityanalytics": "1.0.0",
-      "com.unity.modules.unitywebrequest": "1.0.0",
-      "com.unity.modules.unitywebrequestassetbundle": "1.0.0",
-      "com.unity.modules.unitywebrequestaudio": "1.0.0",
-      "com.unity.modules.unitywebrequesttexture": "1.0.0",
-      "com.unity.modules.unitywebrequestwww": "1.0.0",
-      "com.unity.modules.vehicles": "1.0.0",
-      "com.unity.modules.video": "1.0.0",
-      "com.unity.modules.vr": "1.0.0",
-      "com.unity.modules.wind": "1.0.0",
-      "com.unity.modules.xr": "1.0.0"
-    },
-    "testables": [
-      "com.unity.multiplayer.mlapi"
-    ]
-  }
-  
\ No newline at end of file
+  "dependencies": {
+    "com.unity.collab-proxy": "1.3.9",
+    "com.unity.ide.rider": "2.0.7",
+    "com.unity.ide.visualstudio": "2.0.7",
+    "com.unity.ide.vscode": "1.2.3",
```

Comment:
changes in this file (under testproject) are due to 2019.4 to 2020.3 upgrade. package versions are being updated here.